src/main/java/ru/spbstu/reader/VcfReader.java

```diff
@@ -0,0 +1,27 @@
+package ru.spbstu.reader;
+
+import htsjdk.variant.vcf.VCFIterator;
+import htsjdk.variant.vcf.VCFIteratorBuilder;
+import ru.spbstu.reader.dto.VcfRecord;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class VcfReader {
```

Comment:
If it is util class then make this class final and add a private constructor with comment `//not instatiable`

Reply:
If it is a utility class it would be enough to use Lombok annotation
`@NoArgsConstructor(access = AccessLevel.PRIVATE)`
But I'm not sure that its a ideologically utility class. I assume it's just a reader with static methods.

Reply:
It's not necessary to make the class final, because I assume we will mock it in our test.
But agree about constructor, forgot about this.

Reply:
maybe just @UtilityClass?

Reply:
In  cases when we will mock VcfReader.read (I know that it's not the best idea to mock static method) we would have some issues due to final modiicator